tests/test_models/deterministic.py

```diff
@@ -171,35 +171,34 @@ def absorbing_retirement_filter(retirement, lagged_retirement):
 
 
 @dataclass
-class DiscreteConsumptionStatus:
-    low: int = 1
-    high: int = 2
-
-
-DiscreteWealthStatus = make_dataclass(
```

Comment:
What do we gain and loose by using a continuous grid? That is, an alternative would be to just use, say, five discrete grid points and keep the example *fully* discrete?

Reply:
Went _fully_ discrete.